assets/templates/main.tmpl

```diff
@@ -18,10 +18,15 @@
     <meta name="apple-mobile-web-app-status-bar-style" content="#58595B">
     {{ if .Metadata.ServiceName }}<meta name="ons:service" content="{{ .Metadata.ServiceName}}">{{ end }}
 
-    {{ if .FeatureFlags.ONSDesignSystemVersion }}
-      <link rel="stylesheet" href="https://cdn.ons.gov.uk/sdc/design-system/{{.FeatureFlags.ONSDesignSystemVersion}}/css/main.css">
+    {{ if .FeatureFlags.SixteensVersion }}
+      {{ if eq .SiteDomain "localhost" }}
+        <link rel="stylesheet" href="http://localhost:9000/dist/css/main.css">
+      {{ else }}
+        <link rel="stylesheet" href="http://cdn.ons.gov.uk/sixteens/{{.FeatureFlags.SixteensVersion}}/css/main.css">
```

Comment:
Minor styling comment, you have a space between the handlebars and the content elsewhere

ie. {{ .SomeContent }}